mail-filter/rspamd/rspamd-1.9.4.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Is there anything blocking EAPI 7?

Reply:
I don't think so, but I'll leave that to Christian. I have made the other changes you requested, though.